examples/validation/main.go

```diff
@@ -0,0 +1,50 @@
+// basic example illustrates how to build a very simple baker-based program with just
+// input and output components
+package main
+
+import (
+	"log"
+	"strings"
+
+	"github.com/AdRoll/baker"
+	"github.com/AdRoll/baker/filter"
+	"github.com/AdRoll/baker/input"
+	"github.com/AdRoll/baker/output"
+)
+
+func main() {
+	toml := `
+[fields]
+	names=["timestamp", "source", "target"]
+
+[validation]
+	timestamp="^value[0-9]+$"
+	target="value3"
```

Comment:
As explained above (in e2e_test.go comment) we should generate a validation function which makes at least one record _not pass_ the check